manifests/0000_05_config-operator_01_scheduling.crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: scheduling.config.openshift.io
+spec:
+  group: config.openshift.io
+  scope: Cluster
+  names:
+    kind: Scheduling
+    singular: scheduling
+    plural: scheduling
```

Comment:
doing this means that you'll need to special case yourself in the client generator like https://github.com/openshift/origin/pull/22126